app/assets/stylesheets/govuk_publishing_components/components/_attachment-link.scss

```diff
@@ -1,4 +1,12 @@
+@import "govuk_publishing_components/individual_component_support";
```

Comment:
Nitpick: would you be able to add a little more detail about this change in the commit message? For example, in this commit I'd write something like:

```
Improve attachment link component print styles

- add individual_component_support to allow `govuk-media-query` to work
- set font size to inherit because...
```

Reply:
Commit description updated as requested. Note that I made a couple of changes to this commit compared to what you previously reviewed.